### PR TITLE
chore: Move sdks to top level

### PR DESCRIPTION
**Pull request by cprice404**
The thrust of this PR is to remove the "SDKs" sections from under the "Develop" menu for each product; this brings us back to a world where we only have one "SDKs" page rather than a copy for every language for every product. So this would unify them back into a single page per SDK language rather than separate ones for Cache/Topic/etc.

The nav to reach the SDKs page is now in the top right, next to "Blog", etc. This is what @allenheltondev and I saw for other docs sites that we admire.

There is a new splash page for SDKs, which contains button links for relevant things for each SDK.

### CODE REVIEW

**Comment by cprice404** [comment]
partial screenshot of what the current SDK landing page looks like. (don't laugh!)

image

File: src/components/LinkButton/LinkButton.module.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.link_button {
```
Comment (cprice404) [inline src/components/LinkButton/LinkButton.module.css]: will need help with styling here

File: docs/sdks/index.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
+---
```
Comment (cprice404) [inline docs/sdks/index.md:1]: this is the page that I need the most input/feedback on.

**Comment by allenheltondev** [comment]
I updated this PR with button styling and the link to the SDKs in top nav
image

**Review by allenheltondev — APPROVED** [review]

**Pull request merged**